roles/edpm_ovn_bgp_agent/meta/argument_specs.yml

```diff
@@ -3,4 +3,127 @@ argument_specs:
   # ./roles/edpm_ovn_bgp_agent/tasks/main.yml entry point
   main:
     short_description: The main entry point for the edpm_ovn_bgp_agent role.
-    options: {}
+    options:
+      edpm_ovn_bgp_agent_enable:
+        type: bool
+        default: true
+        description: Enable the bgp agent or not?
+      edpm_ovn_bgp_agent_debug:
+        type: bool
+        default: true
+        description: Start the agent in a debug mode?
+      edpm_ovn_bgp_agent_reconcile_interval:
+        description: Reconciliation interval in seconds
+        type: int
+        default: 300
+      edpm_ovn_bgp_agent_expose_tenant_networks:
+        description: Exposing tenant networks.
+        type: bool
+        default: false
+      edpm_ovn_bgp_agent_expose_ipv6_gua_tenant_networks:
+        description: Exposing tenant networks with IPv6 addresses.
+        type: bool
+        default: false
+      edpm_ovn_bgp_agent_driver:
+        type: str
+        default: ovn_bgp_driver
+      edpm_ovn_bgp_agent_private_key:
+        description: Location of private TLS key. Keep it safe.
+        type: path
+        default: /etc/pki/tls/private/ovn_bgp_agent.key
+      edpm_ovn_bgp_agent_certificate:
+        description: Location of TLS certificate used by BGP agent.
+        type: path
+        default: /etc/pki/tls/certs/ovn_bgp_agent.crt
+      edpm_ovn_bgp_agent_ca_cert:
+        description: Location of IPA certificate.
+        type: path
+        default: /etc/ipa/ca.crt
+      edpm_ovn_bgp_agent_internal_tls_enable:
+        description: Enable internal TLS.
+        type: bool
+        default: false
+      edpm_ovn_bgp_agent_config_basedir:
+        description: Location of BGP agent configuration base directory.
+        type: path
+        default: "/var/lib/config-data/ansible-generated/ovn-bgp-agent"
+      edpm_ovn_bgp_agent_bgp_as:
+        description: BGP Autonomous System number
+        type: int
+        default: 64999
+      edpm_ovn_bgp_agent_clear_vrf_routes_on_startup:
+        type: bool
+        default: false
+      edpm_ovn_bgp_agent_bgp_nic:
+        type: str
+        default: bgp-nic
+      edpm_ovn_bgp_agent_bgp_vrf:
+        type: str
+        default: bgp-vrf
+      edpm_ovn_bgp_agent_bgp_vrf_table_id:
+        description: ID of Virtual Routing and Forwarding table.
```

Comment:
`description: The Routing Table ID that the VRF should use. If it does not exist, this table will be created.`